application-analytics-ui/src/main/resources/Analytics/Code/Macros/MostViewedPages.xml

```diff
@@ -0,0 +1,799 @@
+<?xml version="1.1" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.4" reference="Analytics.Code.Macros.MostViewedPages" locale="">
+  <web>Analytics.Code.Macros</web>
+  <name>MostViewedPages</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <parent>Analytics.Code.Macros.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>MostViewedPages</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>{{mostViewedPages  startDate="2023/07/01 13:37" endDate="2023/08/01 16:37"/}}
+
+{{mostViewedPages  startDate="2023/07/01 13:37" endDate="2023/07/14 16:37"/}}</content>
```

Comment:
You should not have this date values committed. To still have an example on the macro page, you could add only one macro call, without date fields since these might not be relevant for the user accessing the page
This adds another question: since date parameters are not mandatory, you should also treat this case in your code (make the date-range boundlessness)